pkg/apic/subscription.go

```diff
@@ -363,7 +363,9 @@ func (s *CentralSubscription) setAPIResourceInfo(apiServerResource *v1.ResourceI
 	stage, _ := util.GetAgentDetailsValue(apiServerResource, defs.AttrExternalAPIStage)
 	s.RemoteAPIID = apiID
 	s.RemoteAPIStage = stage
-	s.RemoteAPIAttributes = apiServerResource.Attributes
+
+	// get the x agent details for this, convert to map[string]string
+	s.RemoteAPIAttributes = util.MapStringInterfaceToStringString(util.GetAgentDetails(apiServerResource))
```

Comment:
How come we need this?

Reply:
This is identifiable information for associating an api on the dataplane to the subscription from the UC. If the agent, such as streams, adds more agent details then they want to get to it not the actual attributes of the instance.